custom_components/tuya_local/devices/poolex_arctic_action.yaml

```diff
@@ -0,0 +1,268 @@
+name: Pool heatpump
+products:
+  - id: ea8w1w
+    name: Poolex Arctic Action
+primary_entity:
+  entity: climate
+  icon: "mdi:hot-tub"
+  translation_key: pool_heatpump
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: Heat_Intelligent
+              value: heat
+            - dps_val: Heat_Powerful
+              value: h_powerful
```

Comment:
hvac_mode values must be from the predefined list https://developers.home-assistant.io/docs/core/entity/climate/#hvac-modes (the constants listed there map to all lower case strings matching the name of the constant).

In the case of multiple presets matching an hvac_mode, the full set will be accessible through preset_mode, and here all the heating values should map to heat (mark others hidden so that the  preset_mode that is selected when switching to heat mode is deterministic) and the cooling related ones to cool.